common/src/main/java/com/wynntils/features/overlays/FixCrosshairPositionFeature.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright © Wynntils 2022-2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.overlays;
+
+import com.mojang.blaze3d.platform.GlStateManager;
+import com.mojang.blaze3d.systems.RenderSystem;
+import com.wynntils.core.components.Managers;
+import com.wynntils.core.consumers.features.Feature;
+import com.wynntils.core.consumers.overlays.OverlayPosition;
+import com.wynntils.core.consumers.overlays.SectionCoordinates;
+import com.wynntils.core.persisted.config.Category;
+import com.wynntils.core.persisted.config.ConfigCategory;
+import com.wynntils.mc.event.RenderEvent;
+import com.wynntils.utils.mc.McUtils;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.Gui;
+import net.neoforged.bus.api.EventPriority;
+import net.neoforged.bus.api.SubscribeEvent;
+
+@ConfigCategory(Category.OVERLAYS)
```

Comment:
Isn't this more of an UTILITY than an OVERLAY?

Reply:
However, this begs the question, whether we should fix this, or just make users use separate mods, if they care to

Reply:
I think that's why it should move to the utilities category, because the OVERLAY category is strictly reserved for our custom overlays.